app/views/tutorials/new.html.erb

```diff
@@ -13,6 +13,15 @@
         <%= f.label :tagline %>
         <%= f.text_field :tagline, class: 'form-control' %>
 
+        <%= f.label :author %>
+        <%= f.text_field :author, class: 'form-control' %>
```

Comment:
My thinking behind this was a lot of tech helpers may reference material that is not their own, so if they are part of a research team they may want to use the main researchers name instead of their own.